src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -346,6 +347,17 @@ bool PerformInteractionRequest::ProcessVRResponse(
     return false;
   }
 
+  if ((InteractionMode::VR_ONLY == interaction_mode_ ||
+       InteractionMode::BOTH == interaction_mode_) &&
+      vr_result_code_ == Common_Result::SUCCESS) {
```

Comment:
@YarikMamykin I believe you could make two separate Boolean variables and compare them. For example:
```
bool is_mode_to_close = (InteractionMode::VR_ONLY == interaction_mode_) || (InteractionMode::BOTH == interaction_mode_);
bool success_result = vr_result_code_ == Common_Result::SUCCESS;
if(is_mode_to_close && success_result)
{...}
```

Reply:
@IGapchuk , fixed in a950e20